<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>630</width>
    <height>370</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>630</width>
    <height>370</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>630</width>
    <height>370</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>426</x>
      <y>210</y>
      <width>141</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTextEdit" name="output">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>173</y>
      <width>256</width>
      <height>168</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnInput">
    <property name="geometry">
     <rect>
      <x>349</x>
      <y>30</y>
      <width>124</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>输入起点</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbAlgorithm">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>66</y>
      <width>122</width>
      <height>30</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>A*算法</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>全局择优</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>宽度优先</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="cbHeuristicFunc">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>102</y>
      <width>122</width>
      <height>30</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>启发函数1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>启发函数2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>启发函数3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>启发函数4</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="btnRandom">
    <property name="geometry">
     <rect>
      <x>481</x>
      <y>66</y>
      <width>124</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>随机重排</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbMultiple">
    <property name="geometry">
     <rect>
      <x>482</x>
      <y>102</y>
      <width>122</width>
      <height>30</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>1倍</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2倍</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3倍</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4倍</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5倍</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6倍</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="btnSearch">
    <property name="geometry">
     <rect>
      <x>349</x>
      <y>136</y>
      <width>124</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>搜索求解</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDisplay">
    <property name="geometry">
     <rect>
      <x>481</x>
      <y>136</y>
      <width>124</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>演示路径</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnInput2">
    <property name="geometry">
     <rect>
      <x>481</x>
      <y>30</y>
      <width>124</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>输入终点</string>
    </property>
   </widget>
   <zorder>layoutWidget</zorder>
   <zorder>label</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
